proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -336,6 +336,18 @@ message RecordChildExecutionCompletedRequest {
 message RecordChildExecutionCompletedResponse {
 }
 
+message VerifyChildExecutionCompletionRecordedRequest {
+    string namespace_id = 1;
+    temporal.api.common.v1.WorkflowExecution workflow_execution = 2;
```

Comment:
what about parent_workflow_execution and child_workflow_execution

Reply:
Updated to `parent_execution` and `child_execution`.